errors/site-shutdown.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+		<title>Maintenance</title>
```

Comment:
```suggestion
		<title>Site Unavailable</title>
```

Reply:
Good question about 402 cacheability. I don't see any special rules for 4xx, aside from 404 which is cached for 1 minute, so I think our normal cache logic will be in play.